From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MrPowerGamerBR <git@mrpowergamerbr.com>
Date: Sat, 1 Jun 2024 01:22:41 -0300
Subject: [PATCH] SPARKLYPOWER Remap SparklyPower hacky legacy NBT tags

This is only useful for us in SparklyPower, but yeah...

diff --git a/ca/spottedleaf/dataconverter/minecraft/converters/itemstack/ConverterItemStackToDataComponents.java b/ca/spottedleaf/dataconverter/minecraft/converters/itemstack/ConverterItemStackToDataComponents.java
index 2d29d89cc45866822189a62bffbe1a8fe57c477b..688230454b42fd7da72833ca8fe631e6fba06eb9 100644
--- a/ca/spottedleaf/dataconverter/minecraft/converters/itemstack/ConverterItemStackToDataComponents.java
+++ b/ca/spottedleaf/dataconverter/minecraft/converters/itemstack/ConverterItemStackToDataComponents.java
@@ -1221,6 +1221,7 @@ public final class ConverterItemStackToDataComponents {
             ret.setString("id", this.id);
             ret.setInt("count", this.count);
             if (!this.tag.isEmpty()) {
+                net.sparklypower.sparklypaper.LegacyNBTRemapper.INSTANCE.remap(this.tag); // SparklyPaper - Remap SparklyPower hacky legacy NBT tags
                 this.components.setMap("minecraft:custom_data", this.tag);
             }
 
